funnel/views/account.py

```diff
@@ -79,14 +79,29 @@ def password_policy_check():
     policy_form.form_nonce.data = policy_form.form_nonce.default()
 
     if policy_form.validate_on_submit():
-        tested_password = password_policy.password(policy_form.candidate.data)
-        failed_tests = tested_password.test()
+        tested_password = password_policy.test_password(
+            policy_form.candidate.data,
+            user_inputs=(
+                [current_auth.user.fullname, current_auth.user.email]
```

Comment:
Shouldn't this be `str(current_auth.user.email)`? Also, it may be None. Also, email claims.